cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -88,6 +96,31 @@ def _check_command_id(self):
             visited_plans = set()
             self._check_recursive_plan(line.command_id, visited_plans)
 
+    @api.depends("command_id", "action")
+    def _compute_command_code(self):
```

Comment:
No need to compute anything. Just hide this field if command action is not SSH or Python.